projects/poirazi_neuron_model/run_tm_dim_experiment.py

```diff
@@ -0,0 +1,157 @@
+# ----------------------------------------------------------------------
```

Comment:
Why are these TM files in `poirazi_neuron_model`?  Do you want to just rename the directory to `sdr_paper_neuron_models` or something like that?